### PR TITLE
Improve Access Key

### PR DESCRIPTION
**Pull request by geof90**
Generate a better access key that has the account ID mixed into it.

### CODE REVIEW

**Comment by silhouettes** [comment]
I implemented some of the changes we talked about, is that okay with you? Basically, the intention is to use the guid for uniqueness, and the crypto random string for unpredictability.

When the two are put together and base64-encoded, the key is 40 characters long. I am tempted to just remove the accountId at the end now, because I imagine it might be easy to confuse your keys with each other if they all end with the same characters, and I'm not sure if the accountId at the end achieves anything. What do you think?

File: sdk/script/management/account-manager.ts [hunk 1]
```diff
@@ -73,7 +74,7 @@ export class AccountManager {
                 req = req.send({ identity: "accessKey" })
```
Comment (silhouettes) [inline sdk/script/management/account-manager.ts]: I noticed that the 'if' in the collapsed section above still has 'Login the old way' as a case. I think we can just remove it, seeing as all of the keys will be migrated?

Comment (geof90) [inline sdk/script/management/account-manager.ts]: Keys will be migrated on the server side, but on the client side, people who used the CLI prior to the change will have them persisted the old way. The format we are persisting them is different, and this code handles that.

**Comment by silhouettes** [comment]
Okay sorry for the hassle, thanks for discussing it with me. Is this okay with you?

**Comment by geof90** [comment]
LGTM. Thanks for taking this over! :)

**Pull request merged**